src/index.ts

```diff
@@ -32,7 +57,27 @@ export default function publish(options: PublishOptions = {}): Promise<PublishRe
       return;
     }
 
-    const cmd = `npm publish ${args.join(' ')}`;
+    if (tagCheck) {
```

Comment:
Massive nit, could we call this `shouldCheckDistTag` for consistency with `hasDistTag`?

Maybe including `dist-tag` in `--no-tag-check` too? (i.e. `--no-dist-tag-check`)

Reply:
Actually just looked at npm docs, lets just remove `dist` from `hasDistTag` for consistency with `npm publish` e.g.
- `shouldCheckTag`
- `hasTag`
- `no-tag-check`

Reply:
Done!

Reply:
Gah, should have refreshed! One moment...

Reply:
Now done! 😅